README.md

```diff
@@ -46,7 +46,7 @@ The recommended way to use this library is to consume it from maven central whil
   <dependency>
       <groupId>software.amazon.msk</groupId>
       <artifactId>aws-msk-iam-auth</artifactId>
-      <version>1.1.0</version>
+      <version>1.1.4</version>
```

Comment:
`1.1.5`?

Reply:
`1.1.4` is the latest release from what I see. This should be bumped to `1.1.5` once `1.1.5` is released